src/search/data_loader.cpp

```diff
@@ -119,6 +119,49 @@ std::vector<ahItem*> CDataLoader::GetAHItemsToCategory(uint8 AHCategoryID, int8*
             ItemList.push_back(PAHItem);
         }
     }
+
+    if (settings::get<bool>("search.OMIT_NO_HISTORY"))
+    {
+        const char* noQtyQuery = "SELECT item_basic.itemid, item_basic.stackSize "
+                                 "FROM item_basic "
+                                 "LEFT JOIN auction_house ON item_basic.itemId = auction_house.itemid "
+                                 "LEFT JOIN item_equipment ON item_basic.itemid = item_equipment.itemid "
+                                 "LEFT JOIN item_weapon ON item_basic.itemid = item_weapon.itemid "
+                                 "WHERE aH = %u AND auction_house.itemid IS NOT NULL "
+                                 "GROUP BY item_basic.itemid "
+                                 "%s";
+
+        int32 noQtyRet = sql->Query(noQtyQuery, AHCategoryID, OrderByString);
+        if (noQtyRet != SQL_ERROR && sql->NumRows() != 0)
+        {
+            while (sql->NextRow() == SQL_SUCCESS)
+            {
+                uint16 CurrItemID = sql->GetUIntData(0);
+                bool   itemFound  = false;
+                for (ahItem* PAHItem : ItemList)
+                {
+                    if (PAHItem->ItemID == CurrItemID)
+                    {
+                        itemFound = true;
+                        break;
+                    }
+                }
```

Comment:
failed to notice the lack of line break here before I reflex-approved.

Reply:
Hi Teo, I'm not clear on what change I need to make here, can you please explain? :)

Reply:
new line between the closing brace and the if that follows. not sure why style checks didn't complain.

Reply:
```
}
if (stuff)
{
    moreStuff()l
}
```
vs 
```
}

if (stuff)
{
    moreStuff()l
}
```